src/rust/madara/template/src/template.rs

```diff
@@ -77,12 +82,16 @@ impl Default for MadaraSiteData {
 			chapter_selector: String::from("li.wp-manga-chapter"),
 			// a to get the base id from requests to admin-ajax.php
 			base_id_selector: String::from("h3.h5 > a"),
+			// chapter date format
+			date_format: String::from("MMM d, yyyy"),
 			// div to select images from a chapter
 			image_selector: String::from("div.page-break > img"),
 			// div to select all the genres
 			genre_selector: String::from("div.genres-content > a"),
 			// choose between two options for chapter list POST request
 			alt_ajax: false,
+			// use user agent for all http requests
+			use_user_agent: false,
```

Comment:
instead of having a bool to use one built in `USER_AGENT`, this should be a `Option<String>` that defaults to `None`